utils/gen_onnx_mlir.py

```diff
@@ -369,6 +369,7 @@
     'GreaterOrEqual',
     'Hardmax',
     'If',
+    'IsInf'
```

Comment:
@tungld I did add it to `utils/gen_onnx_mlir.py` and ran the `make OMONNXOpsIncTranslation` command but that did not work.

Reply:
I am going to try to rebuild the entire build directory to see if the changes will pick up then

Reply:
Okay I was missing a `,` and that is why the ONNXOps.td.inc file was not updating. Fixed now :)